src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 15
+#define SOF_ABI_MINOR 16
```

Comment:
@slawblauciak Lets make this 15 so we dont need to block. I will update the ABI table.

Reply:
NAK that, kernel is already merged. Will change back to 16 and others to 16.